### ISSUE TITLE
Can't set object definition variables to `var/const/` variables

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#179]
Needed for NTstation.

Example

```
var/const/RADIO_FREQ = 1234

/obj/item/radio/common
    frequency = RADIO_FREQ
```

**Comment by ike709** [comment]
There seems to be a variety of issues with consts but here's a somewhat-related stack trace:
```
Unhandled exception.Unhandled exception. System.Exception: expression DMCompiler.DM.Expressions.Field can not be const-evaluated
   at DMCompiler.DM.DMExpression.ToConstant() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\DMExpression.cs:line 46
   at DMCompiler.DM.DMExpression.Constant(DMObject dmObject, DMProc proc, DMASTExpression expression) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\DMExpression.cs:line 41
   at DMCompiler.DM.Visitors.DMVisitorProcBuilder.VisitProcStatementSwitch(DMASTProcStatementSwitch statementSwitch) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorProcBuilder.cs:line 29
0
   at OpenDreamShared.Compiler.DM.DMASTProcStatementSwitch.Visit(DMASTVisitor visitor) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamShared\Compiler\DM\DMAST.cs:line 538
   at DMCompiler.DM.Visitors.DMVisitorProcBuilder.VisitProcBlockInner(DMASTProcBlockInner block) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorProcBuilder.cs:line 46
   at OpenDreamShared.Compiler.DM.DMASTProcBlockInner.Visit(DMASTVisitor visitor) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamShared\Compiler\DM\DMAST.cs:line 164
   at DMCompiler.DM.Visitors.DMVisitorProcBuilder.VisitProcStatementIf(DMASTProcStatementIf statement) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorProcBuilder.cs:line 131
   at OpenDreamShared.Compiler.DM.DMASTProcStatementIf.Visit(DMASTVisitor visitor) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamShared\Compiler\DM\DMAST.cs:line 400
   at DMCompiler.DM.Visitors.DMVisitorProcBuilder.VisitProcBlockInner(DMASTProcBlockInner block) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorProcBuilder.cs:line 46
   at OpenDreamShared.Compiler.DM.DMASTProcBlockInner.Visit(DMASTVisitor visitor) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamShared\Compiler\DM\DMAST.cs:line 164
   at DMCompiler.DM.Visitors.DMVisitorProcBuilder.VisitProcDefinition(DMASTProcDefinition procDefinition) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorProcBuilder.cs:line 38
   at OpenDreamShared.Compiler.DM.DMASTProcDefinition.Visit(DMASTVisitor visitor) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamShared\Compiler\DM\DMAST.cs:line 209
   at DMCompiler.DM.DMProc.Compile(DMObject dmObject) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\DMProc.cs:line 58
   at DMCompiler.DM.DMObject.CompileProcs() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\DMObject.cs:line 37
   at DMCompiler.DM.Visitors.DMVisitorObjectBuilder.BuildObjectTree(DMASTFile astFile) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\DM\Visitors\DMVisitorObjectBuilder.cs:line 18
   at DMCompiler.Program.Compile(List`1 preprocessedTokens) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\Program.cs:line 93
   at DMCompiler.Program.Main(String[] args) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\DMCompiler\Program.cs:line 22
   ```

**Comment by wixoaGit** [comment]
Fixed by ed15820c26709eed1db62a529becd99e35bcec2a.
